### ISSUE TITLE
Focus not visible

### ISSUE THREAD

**Issue by meldafert** [Grsmto/simplebar#676]
**Describe the bug**
When navigating content with the keyboard with the TAB key, all elements receiving focus should indicate this eg. by showing a visible focus ring.
Since https://github.com/Grsmto/simplebar/pull/587, the scrollable area can be focused with the keyboard, however it does not have a visible focus indicator.

**To Reproduce**
1. Go to the [Demo page](https://grsmto.github.io/simplebar/examples/)
2. Use the TAB key (and SHIFT+TAB) to navigate around the page
3. See that the buttons receive a dotted outline when receiving focus
4. See that whenever the scrollable container is focus, the focus indicator seemingly vanishes

**Expected behavior**
It should always be visible which element has focus. When the scrollable container has focus, it should have a visible indicator.

**Additional context**
[WCAG Success Criterion 2.4.7](https://www.w3.org/TR/WCAG21/#focus-visible)

**Your environment**

| Software         | Version(s) |
| ---------------- | ---------- |
| SimpleBar       | 6.2.5
| Browser          | Firefox 121
| npm/Yarn         | N/A
| Operating System | Linux

**Comment by kalnode** [comment]
Is it actually desired to show a focus ring around a scrollable area?

What happens natively?

**Comment by deltragon** [comment]
If it is focusable, it should have a focus ring.
Firefox, which is the only browser currently to make scrollable areas focusable like default (like simplebar does), does show a focus ring when focusing via keyboard.
See [this article](https://adrianroselli.com/2022/06/keyboard-only-scrolling-areas.html) for more background on the accessibility issues.

**Comment by meldafert** [comment]
If it is focusable, it should have a focus ring.
Firefox, which is the only browser currently to make scrollable areas focusable like default (like simplebar does), does show a focus ring when focusing via keyboard.
See [this article](https://adrianroselli.com/2022/06/keyboard-only-scrolling-areas.html) for more background on the accessibility issues.
